orbs/pip-audit.yml

```diff
@@ -0,0 +1,119 @@
+version: 2.1
+orbs:
+    utils: arrai/utils@1.21.1
+executors:
+    python313:
+        docker:
+            - image: cimg/python:3.13
+    python312:
+        docker:
+            - image: cimg/python:3.12
+    python311:
+        docker:
+            - image: cimg/python:3.11
+    python310:
+        docker:
+            - image: cimg/python:3.10
+    python39:
+        docker:
+            - image: cimg/python:3.9
+aliases:
+    common_parameters: &common_parameters
+        executor:
+            description: "The executor to use for the job."
+            type: executor
+            default: python313
+        resource_class:
+            description: "The resource class to use for the job."
+            type: enum
+            enum: [small, medium, medium+, large, xlarge, 2xlarge, 2xlarge+]
+            default: small
+        config:
+            description: "Any additional configuration steps."
+            type: steps
+            default: []
+        setup:
+            description: "Any additional setup steps."
+            type: steps
+            default: []
+        env_type:
+            description: "Python environment type."
+            type: enum
+            default: "pipenv"
+            enum: [pip, pipenv, uv]
+        create_badges:
+            description: "Generate badges."
+            type: boolean
+            default: true
+        ignored_vulnerabilities:
+            description: "Space separated list of ignored vulnerabilities, e.g. 'CVE-2025-27516'"
+            type: string
+            default: ""
+    common_steps: &common_steps
+        - checkout
+        - steps: <<parameters.setup>>
+        - run:
+              name: Install requirements
+              command: |
+                  if [ "<<parameters.env_type>>" != "uv" ]; then
+                      pip install --upgrade pip-audit pipenv
+                  else
+                      if ! [ -x "$(command -v uv)" ]; then
+                          curl -LsSf https://astral.sh/uv/install.sh | bash
+                      else
+                          uv self update
+                      fi
+                  fi
+        - steps: <<parameters.config>>
+        - run:
+              name: Run pip-audit
+              command: |
+                  IGNORES=(<<parameters.ignored_vulnerabilities>>)
+                  ARGS=("${IGNORES[@]/#/--ignore-vuln }")
```

Comment:
Do you want to ignore empty ignored? Mr. AI Overlord also suggests `read` for array safety.

```bash
if [ -n "<<parameters.ignored_vulnerabilities>>" ]; then
    read -ra IGNORES <<< "<<parameters.ignored_vulnerabilities>>"
    ARGS=("${IGNORES[@]/#/--ignore-vuln }")
fi
```